presto-hive/src/main/java/com/facebook/presto/hive/HivePartitionManager.java

```diff
@@ -185,7 +185,7 @@ public Iterable<HivePartition> getPartitionsIterator(
                         return result.iterator();
                     }
                     catch (InterruptedException | ExecutionException e) {
-                        log.error("Parallel parsing of partition values failed with error message:%s", e.getMessage());
+                        log.error(e, "Parallel parsing of partition values failed with error message:%s", e.getMessage());
```

Comment:
you can remove `e.getMessage()` since you are now logging the entire stacktrace.